src/print_air.zig

```diff
@@ -350,6 +351,15 @@ const Writer = struct {
         try s.print(", {s}", .{@tagName(atomic_load.order)});
     }
 
+    fn writePrefetch(w: *Writer, s: anytype, inst: Air.Inst.Index) @TypeOf(s).Error!void {
+        const prefetch = w.air.instructions.items(.data)[inst].prefetch;
+
+        try w.writeOperand(s, inst, 0, prefetch.ptr);
+        try s.print(", {s}, {d}, {s}", .{
+            @tagName(prefetch.rw), prefetch.locality, @tagName(prefetch.cache),
```

Comment:
`__builtin_prefetch()` under gcc and clang doesn't accept a cache argument, only address, rw, and locality. If cache is not `.data` we this should be a noop.

See: https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html